service-base/src/main/java/org/eclipse/hono/service/http/DefaultFailureHandler.java

```diff
@@ -75,6 +76,32 @@ public void handle(final RoutingContext ctx) {
         }
     }
 
+    private String extractMessage(final HttpServerResponse response) {
+        if ( response == null ) {
+            return MSG_NO_ERROR;
+        }
+
+        final String message = response.getStatusMessage();
+        if ( message == null ) {
+            return Integer.toString(response.getStatusCode());
```

Comment:
this information is already given by the response's status code. I would rather use the empty string here again (i.e. an empty body)

Reply:
ok